### PR TITLE
feat(curriculum): adding the Mathbot workshop to frontend cert

### PR DESCRIPTION
**Pull request by jdwilkin4**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/workshop-mathbot/66ea7adae8053065a64f9002.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+title: Step 1
+challengeType: 1
+dashedName: step-1
+---
+
+# --description--
+
+In this workshop, you will review working with the different `Math` object methods by building a Mathbot. This Mathbot will log some math operations and messages to the console in efforts to teach you about JavaScript's `Math` object.
+
+For this first step, start by creating a variable called `botName` and assign the string value of `"MathBot"`.
```
Comment (Dario-DC) [inline curriculum/challenges/english/25-front-end-development/workshop-mathbot/66ea7adae8053065a64f9002.md:12]:
```suggestion
For this first step, start by creating a variable called `botName` and assign it the string value of `"MathBot"`.
```

**Review by Dario-DC — APPROVED** [review]
Left a comment. Other than that, it looks good to me :+1:

File: curriculum/challenges/english/25-front-end-development/workshop-mathbot/66ea7fc048a6e26e33c598fe.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+# --description--
+
+Now, it is time to generate a random number using the `Math.random()` method.
+
+Create a variable called `randomNum` and assign it the value of the result of calling the `Math.random()` method.
+
+Then, log the `randomNum` variable to the console.
+
+Try refreshing the page a few times to see the different random numbers that are generated.
```
Comment (moT01) [inline curriculum/challenges/english/25-front-end-development/workshop-mathbot/66ea7fc048a6e26e33c598fe.md:16]: I'm fine with it, but maybe a different suggestion other than refreshing the page. Maybe "Try adding a space in the code to re-run the bot and see different random numbers that are generated.

File: curriculum/challenges/english/25-front-end-development/workshop-mathbot/66ea845f54195c7020cc521b.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+challengeType: 1
+dashedName: step-5
+---
+
+# --description--
+
+The formula to generate a random number between two values is the following:
+
+```js
+Math.random() * (max - min) + min;
```
Comment (moT01) [inline curriculum/challenges/english/25-front-end-development/workshop-mathbot/66ea845f54195c7020cc521b.md:13]: You could change this example to use maybe `maximum` and `minimum` or `maxNumber` and `minNumber` or something - so people can't just copy/paste the example.

**Review by moT01 — COMMENTED** [review]
LGTM @jdwilkin4 🎉 It was fun 👍

I left a few suggestions that aren't necessary to add. If you're fine with it how it is, let me know and we can merge it.

**Review by moT01 — APPROVED** [review]

**Pull request merged**